platform-coop-toolkit/pcc-framework
# feat: Add authoring feature for Community Stories

## Description
Adds a `pcc-story` post type to the Admin interface and allows the authoring of content to be featured the Community Voices > Stories section of the front-end site.

## To Test

1. Activate the `pcc-framework plugin` with the changes from this pull request.
2. Go to "Stories > Organization", "Stories > Region", "Stories > Sector" and add a new term to each of those taxonomies.
3. Go to "Stories > Add new"
4. Enter a title and a quote in the body.
5. In the sidebar, specify a Youtube video url (ex. https://www.youtube.com/watch?v=savByUiUvbY&feature=youtu.be)
6. Add a Featured Image
7. Choose a Storyteller (it should be populated with names from the Persons post type)
8. Choose a Sector, Region, and Organization (the terms you specified in step 2 should appear in these lists).
9. Publish.

## Expected behaviour
The Story content should save and be listed under "All stories".

## Known issues
Currently there is no input validation. See #162 .
